packages/babel-helpers/src/helpers/defineProperty.ts

```diff
@@ -1,7 +1,13 @@
 /* @minVersion 7.0.0-beta.0 */
+import toPropertyKey from "./toPropertyKey.ts";
 
-import toPropertyKey from "toPropertyKey";
-export default function _defineProperty(obj, key, value) {
+type Object = { [key: symbol | string]: unknown };
+
+export default function _defineProperty<T extends Object>(
+  obj: T,
+  key: PropertyKey,
```

Comment:
Just saw the comment about using `string | symbol`. Will fix it here

Reply:
`defineProperty` is different since `defineProperty` does allow `number` to be a valid object key. Using `string | symbol` is mistyping.